addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -645,6 +643,8 @@ protected synchronized void addDeviceAsChannel(Device device) {
         // Default, depends on device type
 
         if (device != null) {
+            logger.debug("Add virtual device as channel: {}", device.getMetrics().getTitle());
```

Comment:
After seeing a few of these: Was the intention of the original author to always log something, and if so, should there be an additional `if (device == null) logger....` line? Depends on the context I guess.